House Keeping & Security policy (#1072)
digitalfabrik/integreat-cms: SECURITY.md

# Security

## Reporting Potential Security Issues

If you have encountered a potential security vulnerability in this project,
please report it to us at <security@tuerantuer.org>. We will work with you to
verify the vulnerability and patch it.

When reporting issues, please provide the following information:

- Component(s) affected
- A description indicating how to reproduce the issue
- A summary of the security vulnerability and impact

We request that you contact us via the email address above and give the
project contributors a chance to resolve the vulnerability and issue a new
release prior to any public exposure; this helps protect the project's
users, and provides them with a chance to upgrade and/or update in order to
protect their applications.

## Policy

If we verify a reported security vulnerability, our policy is:

> - We will patch the current release branch, as well as branches that are used by our partners.
> 
> - After patching the release branches, we will immediately issue new security
>   fix releases for each patched release branch.

Reviewer: ```suggestion
- We will patch the default branch and immediately issue a new security fix release.
```
Since we're not maintaining multiple release branches and only have a linear, calendar-versioned release scheme, I'd adapt the wording a little bit here.
